src/Simplic.Flow.Editor.UI/ViewModel/WorkflowEditorViewModel.cs

```diff
@@ -547,6 +547,17 @@ public string ServiceName
                 RaisePropertyChanged(nameof(ServiceName));
             }
         }
+
+        public string Description 
+        {
+            get { return flowConfiguration.Description; }
+            set 
+            { 
+                flowConfiguration.Description = value;
+                IsDirty = true;
+                RaisePropertyChanged(nameof(Description));
+            }
+        }
```

Comment:
add comment, and the get should be allignet with the set, so add a new line at the start and the end